webproject/src/main/java/com/google/sps/servlets/MyOrderServlet.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.sps.data.Category;
+import com.google.sps.data.Room;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.annotation.WebServlet;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.io.OutputStream;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+@WebServlet("/myOrder")
```

Comment:
I think this should have roomId in the path. but we can modify later
FYI @bxtsang